genie-core/src/main/java/com/netflix/genie/core/jobs/workflow/impl/GenieBaseTask.java

```diff
@@ -41,6 +42,13 @@
 @Slf4j
 public abstract class GenieBaseTask implements WorkflowTask {
 
+    protected static final String NO_ID_FOUND = "<no id>";
```

Comment:
Don't we already have something like this somewhere? Seems like it would be common across the whole app